graviti/dataframe/frame.py

```diff
@@ -6,6 +6,7 @@
 """The implementation of the Graviti DataFrame."""
 
 
+from collections import defaultdict
```

Comment:
`function` -> `method` in commit message.
normally method means the function in class. function means standalone function